api/api.proto

```diff
@@ -460,6 +460,19 @@ message Stats {
   uint64 interval_ms = 10;
 }
 
+message CapacityUsageInfo {
+  // ExclusiveBytes used by this snap
```

Comment:
Provide enough information on each documentation to explain what it is used for and how to interpret it for those who do not know much about storage.